modules/partition-replicator/src/integrationTest/java/org/apache/ignite/internal/partition/replicator/ItZoneDataReplicationTest.java

```diff
@@ -245,7 +244,13 @@ void testLocalRaftLogReapplication() throws Exception {
      * </ol>
      */
     @Test
-    void testTableDropInTheMiddleOfRebalanceOnOutgoingSide(@InjectExecutorService ExecutorService executorService) throws Exception {
+    void testTableDropInTheMiddleOfRebalanceOnSendingSide(@InjectExecutorService ExecutorService executorService) throws Exception {
+        // Disable automatic low watermark updates.
+        CompletableFuture<Void> cfgChangeFuture = gcConfiguration.lowWatermark()
+                .change(lowWatermarkChange -> lowWatermarkChange.changeUpdateIntervalMillis(Long.MAX_VALUE));
```

Comment:
Would it make sense to extract this to a method? I think it should be used at least twice. Having a method, the comment would become obsolete